apps/builder/app/shared/hook-utils/use-sync-initialize-once.ts

```diff
@@ -1,16 +1,17 @@
-import { useRef } from "react";
+import { useEffect, useRef } from "react";
 
 // This hook allows to
 // 1. run a function only once on the client
-// 2. run it server-side
-// 3. run the function synchronously in both
-// Can't use effect for that (async).
 // Can't use useState initializer because it can be called twice in strict mode and also isn't designed for this.
 export const useSyncInitializeOnce = (fn: () => void) => {
```

Comment:
we need to rename it now, probably to useEffectOnce or useInitializeOnce because its now async